x-pack/docs/en/security/securing-communications/setting-up-ssl.asciidoc

```diff
@@ -19,21 +18,26 @@ applicable to your cluster.
 certificate and enable TLS on the transport layer. You can also optionally
 enable TLS on the HTTP layer. See
 {ref}/configuring-tls.html#tls-transport[Encrypting Communications Between Nodes in a Cluster] and
-{ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications]. 
+{ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications].
 
-. Configure the {monitor-features} to use encrypted connections. See <<secure-monitoring>>.
++
+NOTE: {ref}/configuring-tls.html#tls-http[Encrypting HTTP Client Communications] is strongly recommended if you want to enable TLS encryption for Kibana, Logstash and Beats.
```

Comment:
"Enable TLS for X" might have multiple readings. It could mean "enable X to communicate over https to Elasticsearch", which is what we're trying to convey here I think, but it can also mean "Enable TLS on X itself so that other components can talk to X over TLS" , i.e. browser -> kibana, beats -> logstash etc. 

I think our stance should be that enabling TLS on http is strongly recommended unless there is a very specific reason not to. If we were to frame it as a conditional here, I think we should mention something like : 

```suggestion
NOTE: If you want to enable other stack components like Kibana, Logstash and Beats , or your users via their browsers to communicate securely with Elasticsearch, then {ref}/configuring-tls.html#tls-http[TLS configuration for HTTP] is strongly required so that the communication is encrypted with TLS. 
```

Reply:
I'd make this a stronger statement--completely configuring security should be the recommended path. Something like:

> To enable Kibana, Logstash, and Beats to communicate with Elasticsearch securely, you must configure TLS for HTTP. 

It's not just a suggestion, it's a requirement.

Reply:
I've implemented a combination of these suggestions.